run.py

```
@@ -0,0 +1,813 @@
+bilibili_desired_url = "https://live.bilibili.com/" + config.username
+
+def offline_check(driver, live_url, spare_link, important):
+      refresh_count = 0
+      countdownhours = 0
+      numberpart = 0
+      fewtimes = 0
+      if config.Twitch == "True":
+        try:
+          driver.find_element("xpath", "//button[@data-a-target='content-classification-gate-overlay-start-watching-button']//div[text()='開始觀看']").click()
+          time.sleep(5)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)